pegasus/sites.v3/code.org/public/css/careers.css

```diff
@@ -8,7 +8,8 @@
 .careers__banner,
 .careers__banner--faq {
   align-items: center;
```

Comment:
I removed the values tile related CSS, the rest of the changes here were done automatically by my linter, sorry for the noise.